sbnanaobj/StandardRecord/SRTrigger.h

```diff
@@ -0,0 +1,21 @@
+#ifndef SRTRIGGER_h
+#define SRTRIGGER_h
+
+#include <cstdlib>
+
+namespace caf
+{
+  class SRTrigger
+  {
+  public:
+    SRTrigger();
+    u_int64_t global_trigger_time;
+    u_int64_t beam_gate_time_abs;
+    int64_t trigger_within_gate;
+    double beam_gate_det_time;
+    double global_trigger_det_time;
```

Comment:
It would be good to document the meaning and conventions of these fields, e.g.
```suggestion
    u_int64_t global_trigger_time; ///< Time of trigger in UTC [ns].
    u_int64_t beam_gate_time_abs; ///< Time of beam gate opening in UTC [ns].
    int64_t trigger_within_gate; ///< Time from the beam gate to the trigger [ns]
    double beam_gate_det_time; ///< Beam gate time on detector time scale [us]
    double global_trigger_det_time; ///< Trigger time on detector time scale [us]
```
Note that in CAF [the mandatory unit for times is microseconds](https://github.com/SBNSoftware/sbnanaobj/blob/develop/sbnanaobj/StandardRecord/README.md#unit-conventions) (I argue absolute timestamps are not covered by that convention).